### ISSUE TITLE
Can't create account - Vague message

### ISSUE THREAD

**Issue by rolandog** [davidpanderson/science_united#3]
I've tried to create an account several times. but I'm hit with the following message:
```
Can't create account
Invalid email address: you must enter a valid address of the form name@domain

Click your browser's Back button to try again.
```

Have gmail accounts been blacklisted? Or should I not enter the `.com` part of my email. The error message is a bit vague.

**Comment by davidpanderson** [comment]
Thanks for the report.
What did you enter as an email address?
-- David

**Comment by rolandog** [comment]
Hi David.

I filled in my current personal email address (my current username at gmail.com).

**Comment by davidpanderson** [comment]
You mean rolandog@gmail.com?
I just successfully created (and then deleted) an account with that email.
Check your typing.
-- David

**Comment by rolandog** [comment]
Yes. I deliberately didn't want to type it (so as to avoid spam). Thanks for your suggestion, though. I did check my typing and was still unable to create an account.

Here are some screenshots including the browser console.

![screenshot from 2018-03-19 19-11-04](https://user-images.githubusercontent.com/12400926/37630382-e4bbae60-2ba9-11e8-84bc-d6c4c1c917f3.png)

![screenshot from 2018-03-19 19-13-28](https://user-images.githubusercontent.com/12400926/37630391-ef2e30f2-2ba9-11e8-86f3-d7f538fcfc10.png)

I'm using Firefox Quantum 59.0.1 (64-bit) on Ubuntu 17.10. I had already tried disabling NoScript, uBlock and Disconnect (which are JavaScript blocking add-ins).

I also tested this on Google Chrome Version 62.0.3202.62 (Official Build) (64-bit). Same results.

Finally, I did notice something odd. It appears that there are two different account creation pages. One is the landing page after clicking 'Join' (the one I've been using), and the other is accessed after clicking the 'Create account' link on the 'Login' page. The URLs are, respectively:

* https://scienceunited.org/su_join.php
* https://scienceunited.org/create_account_form.php?next_url=

They appear to ask different questions for the account creation.

I'll try doing the same process but through the second URL. Perhaps there might've been an oversight and the first URL should be deprecated.

**Comment by rolandog** [comment]
I've tried the second URL (in both Firefox and Chrome), and the weird thing is I get the same screen. I'm at a loss as to what is happening. I'm also using a VPN... but I've never had trouble with creating accounts before.

**Comment by davidpanderson** [comment]
I added code that shows the supposedly invalid email addr; please try again.
Your screenshots are very helpful.
-- David

**Comment by davidpanderson** [comment]
I wonder if the VPN is the problem.
Are you able to bypass it?

**Comment by rolandog** [comment]
I'm currently at work; we use a corporate proxy here, and I'm on a Windows 10 machine (which I try to keep up to date).

I tested with the VPN enabled (through the Proxy) and got the same results.

When testing without using a VPN (by using the Proxy), however, I was able to create an account!

I last updated OpenVPN about a month or two ago (on this machine), but since this issue happened on two different machines (with different operating systems), I'm more inclined to say that maybe there's some silent blocking by my provider (NordVPN).

I'll try reaching out to them and pointing them at the account creation page and to this issue, so maybe they can help us find a root cause.

**Comment by davidpanderson** [comment]
Interesting.
Did you try it through the VPN but with a different email address?

**Comment by rolandog** [comment]
I contacted my VPN Provider (NordVPN); they recommended trying different protocols (UDP, IKEv2).

I was able to test the latter (IKEv2) through the work proxy (same results).

I'll test UDP when I get home, as it seems that you can't use OpenVPN with UDP protocol with a proxy.

However, they told me that through UDP they were able to create an account here. As I had some urgent issues at work, I couldn't wait for them to verify with TCP... but I'll contact them again to follow up and also verify if I'm able to create an account with the VPN trhough UDP.
